components/expenses/ExpenseForm.js

```diff
@@ -149,7 +153,10 @@ const validate = expense => {
       : requireFields(expense, ['description', 'payee', 'payee.name', 'payee.email']);
   }
 
-  const errors = isCardCharge ? {} : requireFields(expense, ['description', 'payee', 'payoutMethod', 'currency']);
+  const errors =
+    isCardCharge || [CollectiveType.COLLECTIVE, CollectiveType.ORGANIZATION].includes(expense.payee.type)
```

Comment:
Do we really want to stop requiring a description, a payout method and a currency when the payee is a collective?

Reply:
Ah yes, this is a mistake. I've corrected it. 👍🏽